### PR TITLE
Feat/#124

### PR DESCRIPTION
**Pull request by JankowskiDaniel**

### CODE REVIEW

**Comment by JankowskiDaniel** [comment]
## Important changes

- Each RNN `Predictor` has a new parameter `activation` provided in the config. **If no activation is provided, no activation function is applied to the model output!!!**
- `TverskyLossMultiLabel` has been introduced, fixing a previous bug. Now, Tversky Loss seems to work properly for the classification task. Use `tversky` in the config to use this loss function.
- `test_model` function in `src.test.py` has a new parameter `is_classification` set to `False` by default. This value is automatically assigned based on the config (no manual adjustment required). If the value is set to `True`, the function converts the model outputs to binary values.

## How to run a classification experiment?

1. Initialize the proper config:

```yml
...
  is_classification: True
  class_epsilon: 0.001
training:
  is_curriculum: False
  debug_mode: True
  loss_function: bce_weighted
  encoder:
    name: ShotSeqEncoder
    weights: null
    freeze: False
    learning_rate: 0.001
  predictor:
    name: SimpleCFC
    learning_rate: 0.0005
    weights: null
    hidden_size: 16
    activation: null
...
```
For the classification task, when using either `tversky` or `bce_weighted` loss function, the activation function for the predictor should be set to `null`, because both loss function applies the sigmoid under the hood.

2. Train the model
3. Test the model for the classification: `python ./src/test.py -cp ../results/test_hydra_mse/ hydra.run.dir=. hydra.output_subdir=null hydra/job_logging=disabled hydra/hydra_logging=disabled`. Here, no changes compared to default model testing
4. To test the model for the regression run the following command: `python ./src/test.py -cp ../results/test_hydra_mse/ hydra.run.dir=. hydra.output_subdir=null hydra/job_logging=disabled hydra/hydra_logging=disabled data.is_classification=False training.predictor.activation=relu`. If we want to test the model for the regression, we have to modify the activation function at the end of the predictor to ensure positive values

**Review by kapiblue — APPROVED** [review]

**Comment by kapiblue** [comment]
Running test for the second time will overwrite the results for classification.

**Comment by kapiblue** [comment]
Maybe we should adjust the test script to test regression and classification at the same time?
